src/redisvalue.rs

```diff
@@ -8,12 +13,46 @@ pub enum RedisValue {
     BulkRedisString(RedisString),
     StringBuffer(Vec<u8>),
     Integer(i64),
+    Bool(bool),
     Float(f64),
     Array(Vec<RedisValue>),
+    Map(HashMap<RedisValue, RedisValue>),
+    Set(HashSet<RedisValue>),
```

Comment:
Do we want to support big numbers and attributes?

Reply:
I guess, didn't see the need to add it yet

Reply:
Could be useful for `UInteger(u64)`